application/views/admin/globalsettings/_emailTestConfirmation.php

```diff
@@ -9,10 +9,10 @@
     </div>
 </div>
 <div class="modal-footer">
-    <button role="button" class="btn btn-primary btn-ok" href="<?php echo \Yii::app()->createUrl('admin/globalsettings', array("sa" => "sendTestEmail")); ?>">
+    <a role="button" class="btn btn-primary btn-ok" href="<?php echo \Yii::app()->createUrl('admin/globalsettings', array("sa" => "sendTestEmail")); ?>">
```

Comment:
We can remove the role="button" , no ?

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role